app/src/main/java/com/example/trainingintive/repository/Repository.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.trainingintive.repository
+
+import com.example.trainingintive.network.DogImageUrl
+import io.reactivex.rxjava3.core.Single
+
+interface Repository {
```

Comment:
Keep feature names, in case of need for more "repositories" how you would name it ?